Lib/os.py

```diff
@@ -651,6 +651,29 @@ def get_exec_path(env=None):
     return path_list.split(pathsep)
 
 
+def get_current_dir_name():
+    """
+    Return a string representing the current working directory taking into
+    consideration the users *PWD* environment variable if it exists. This
+    is opposed to getcwd() which dereferences symlinks in the path. This
+    function is identical to getcwd() on systems that do *not* support
+    the *PWD* environment variable.
+    """
+    cwd = getcwd()
+
```

Comment:
I think empty lines in this functions are redundant. Empty lines inside functions are used for separating groups of statements in large functions, but this function is enough small and simple. And will be smaller after removing empty lines.